android/app/build.gradle

```diff
@@ -53,12 +53,12 @@ android {
     }
 
     signingConfigs {
-        debug  {
-            storeFile file('keystore.jks')
-            storePassword 'tech.pylons'
-            keyAlias 'key_tech_pylon'
-            keyPassword 'tech.pylons'
-        }
+//        debug  {
```

Comment:
Is this dead code necessary for testing?

Reply:
It is needed when we release the build.